app/modules/individuals/models.py

```diff
@@ -229,14 +229,59 @@ def merge(cls, *individuals, sex=None, primary_name=None):
                     log, target_individual, f'assigned {enc} from merged {indiv}'
                 )
                 enc.individual_guid = target_individual.guid
-            # TODO also consolidate SocialGroups
+            target_individual._consolidate_social_groups(indiv)
             indiv.delete()
         return result
 
     def merge_from(self, *individuals):
         all_indiv = (self,) + individuals
         return Individual.merge(*all_indiv)
 
+    # mimics Individual.merge() in args, but does not immediately executes; rather waits for approval
+    #   and initiates a request including time-out etc
+    @classmethod
```

Comment:
What's the advantage to making this a class method?
We know the individual that is the target so why not make this a method on the target individual and then rename individuals as from_individuals

Reply:
based on the kind of bumpy history of merging, i strongly suspect that there will be a point when we do not have a _given_ target individual, but rather just an arbitrary list of individuals we want merged.   so i was trying to leave open the option for this signature as a class-method.  "here are N individuals, let the algorithm figure out which one should win and merge the rest into it".

maybe that is too much over-engineering for a future use-case; i can just roll things back to being forced upon a chosen target-individual if so.

Reply:
after discussion with @colinwkingen i am now just chopping off the classmethod flavors of this.  going forward with just `indiv.merge_from()` idea (chosen target-individual) for mvp.